ui/containers/Home/components/__tests__/__snapshots__/PersonalWorkspace.test.tsx.snap

```diff
@@ -71,7 +71,7 @@ exports[`PersonalWorkspace renders correctly 1`] = `
           />
         </div>
         <Connect(Feature)
-          flag="application"
+          flag="applications"
```

Comment:
Do snapshot tests run on the build server?

Reply:
@afoerster Yes, but I think it will only compare props value, as you can see above.